src/boxes/media/sample_table.rs

```diff
@@ -123,85 +125,56 @@ impl Mp4Box for SampleTable {
 
             match name {
                 "stsd" => {
-                    let b = Box::new(InnerAtom::parse(
-                        reader,
-                        index,
-                        level + 1,
-                    )?) as Box<dyn Mp4Box>;
+                    let b =
+                        Box::new(InnerAtom::parse(reader, index, level + 1)?) as Box<dyn Mp4Box>;
                     sample_table.stsd = Some(b);
                 }
                 "stts" => {
-                    let b = Box::new(InnerAtom::parse(
-                        reader,
-                        index,
-                        level + 1,
-                    )?) as Box<dyn Mp4Box>;
+                    let b =
+                        Box::new(InnerAtom::parse(reader, index, level + 1)?) as Box<dyn Mp4Box>;
                     sample_table.stts = Some(b);
                 }
                 "ctts" => {
-                    let b = Box::new(InnerAtom::parse(
-                        reader,
-                        index,
-                        level + 1,
-                    )?) as Box<dyn Mp4Box>;
+                    let b =
+                        Box::new(InnerAtom::parse(reader, index, level + 1)?) as Box<dyn Mp4Box>;
                     sample_table.ctts = Some(b);
                 }
-                "stss" => {
```

Comment:
What happened to this? Why it is removed?

Reply:
It's not removed, it's just re-arranged. See on the right side. This and other re-arrangement are done to print them according to the order specified in the standard ISO document to print them as per them. 

I suppose since this isn't a major change in logic here, can be ignored?